src/fetchers/wakatime-fetcher.js

```diff
@@ -1,14 +1,16 @@
 const axios = require("axios");
 
-const fetchLast7Days = async ({ username }) => {
+const fetchLast7Days = async ({ username, api_domain }) => {
   try {
     const { data } = await axios.get(
-      `https://wakatime.com/api/v1/users/${username}/stats/last_7_days?is_including_today=true`,
+      `https://${
```

Comment:
Hopefully there won't be any XSS issues because of this. we will sanitize these interpolations in future.

Reply:
I don't know how that should be encoded, maybe strip all non "domain safe" characters (regex `/[^a-z-.0-9]/gi` i think)

Reply:
That might work for now.